linux_os/guide/system/software/integrity/crypto/configure_openssl_crypto_policy/ansible/shared.yml

```
@@ -24,7 +24,7 @@
 - name: "Add crypto_policy group and set include openssl.config"
```

small nitpick, please update the name of the file here as well
```suggestion
- name: "Add crypto_policy group and set include opensslcnf.config"
```

---

I have pushed these fixes since they were really simple